working/augmentation-libraries/feature-specification.md

```diff
@@ -575,6 +575,9 @@ declaration, and if not `final`, also an abstract setter declaration. An
 `external` setter.  Unlike abstract declarations, they are considered to
 have a concrete implementation.
 
+Variables which require an initializer expression need not initially be defined
+with one, as long as there exists some augmentation which supplies it.
```

Comment:
It would be good to specify which variables require an initializer. I think it's basically just potentially-non-nullable non-late top-level variables and static fields?

Reply:
It doesn't really feel like the right place to try and enumerate the scenarios, but I will add that as an example, so that we aren't obligated to make it exhaustive or keep it up to date with language changes.

Reply:
I agree, we could just say 'Variables need not initially...'. We could extend that with commentary along the lines of 'Variables need not initially be defined with an initializing expression *(not even non-late variables with a potentially non-nullable type)*, as long...'.